pkg/costattribution/sample_tracker.go

```diff
@@ -67,22 +68,35 @@ func newSampleTracker(userID string, trackedLabels []costattributionmodel.Label,
 		overflowCounter:  observation{},
 	}
 
+	if err := tracker.createDescriptors(trackedLabels); err != nil {
+		return nil, fmt.Errorf("it was impossible to create Prometheus descriptors for cost attribution labels %s configured via -validation.cost-attribution-labels-structured: %w", costattributionmodel.FromCostAttributionLabelsToString(trackedLabels), err)
```

Comment:
Here too we shouldn't talk about the flag IMO, also same feedback about "it was impossible": that applies to all error/log language we're using in this PR.